### PR TITLE
Implement Triangle Exercise

### PR DESCRIPTION
**Pull request by rpalo**
Finish first draft of README

Finish initial test generation
Update tests with skips, fix typos

Create example script
Update config.json with triangle problem data

### CODE REVIEW

File: exercises/triangle/triangle_test.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  run bash triangle.sh equilateral 2 2 2
+
+  [ "$status" -eq 0 ]
+}
+
+@test "false if any side is unequal" {
+  # this is used to skip the test
+  # normally, we skip every test except for the first one
+  # this allows for a person to focus on solving a test at a time
+  # you can comment out or delete the `skip` to run the test when ready
```
Comment (Smarticles101) [inline exercises/triangle/triangle_test.sh]: You can remove these comments :) they're only necessary the first time skips are used in the track.

File: exercises/triangle/example.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+scalene() {
+  assert "$1 != $2" && assert "$1 != $3" && assert "$2 != $3"
+}
+
+# Bash Ternary Operator:
+# (boolean value/calculation) && action if true || action if false
+# Works because of boolean shortcutting.
+# If boolean clause is false, it doesn't evaluate the other side of &&
+# If first thing is false, it evaluates the item after ||,
+# but if first two are true, doesn't bother evaluating last part.
```
Comment (Smarticles101) [inline exercises/triangle/example.sh]: I love all the comments in here. That makes it a lot easier for users unfamiliar with bash to learn by comparing their solution with the provided example

**Comment by Smarticles101** [comment]
@rpalo are you planning on making the change I suggested? If not, let me know and I can finish it for you, or hand it off to @sjwarner-bp to do in #97. Because your pull request came first, yours gets priority.

**Comment by rpalo** [comment]
Hi, yep, sorry, things have been crazy.  I'll knock that out.

**Comment by Smarticles101** [comment]
Thats understandable @rpalo thanks for the work :)

**Pull request merged**